lib/writer/block-read-stream.js

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const stream = require('readable-stream')
+const fs = require('fs')
+const debug = require('debug')('block-read-stream')
+
+/* eslint-disable lodash/prefer-lodash-method */
```

Comment:
Can we have a nested ESLint configuration (in `lib/writer/.eslintrc.yml`) that disables certain rules in the name of supporting low level code?